### PR TITLE
Update README on how to use volumes with docker

### PR DESCRIPTION
**Pull request by metalvegetarianoprogresivo**
Just updated the `README.rm` file showing an example on how to use volumes with docker.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,4 +23,9 @@ docker build -t hubot .
 Run
 ```powershell
 docker run -it --rm -p 8081:8080 --env HUBOT_SLACK_TOKEN="Slack Token" hubot
+```
+
+If you want to use a volume just run:
+```powershell
+docker run -it --rm -p 8081:8080 -v $pwd/scrips:/app/scripts --env HUBOT_SLACK_TOKEN="Slack Token" hubot
```
Comment (cmedinilla) [inline README.md]: Soy yo o hay un typo en el rpimer scripts dice scrips

**Review by cmedinilla — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (metalvegetarianoprogresivo) [reply README.md]: Si lo hay.

Comment (metalvegetarianoprogresivo) [reply README.md]: Ya no.

**Review by cmedinilla — APPROVED** [review]
LGTM

**Pull request merged**